lisa/sut_orchestrator/libvirt/ch_platform.py

```diff
@@ -176,6 +176,29 @@ def _create_node_os_disk(
         self, environment: Environment, log: Logger, node: Node
     ) -> None:
         node_context = get_node_context(node)
+        # if environment.runbook.nodes_requirement:
+        #     node_req: List[schema.NodeSpace] = environment.runbook.nodes_requirement
+        #     for _, node_space in enumerate(node_req):
+        #         node_runbook: BaseLibvirtNodeSchema = node_space.get_extended_runbook(
+        #             self.node_runbook_type(),
+        #             type_name=type(self).type_name(),
+        #         )
+
+        #         if node_runbook.disk_img_resize_gib:
+        #             qemu_tool = self.host_node.tools[QemuImg]
+        #             src_file = node_context.os_disk_base_file_path
+        #             qemu_tool.resize(
+        #                 src_file=src_file,
+        #                 size_gib=node_runbook.disk_img_resize_gib,
+        #             )
+        node_runbook: BaseLibvirtNodeSchema = node.runbook
+        if node_runbook.disk_img_resize_gib:
+            qemu_tool = self.host_node.tools[QemuImg]
+            qemu_tool.resize(
+                src_file=node_context.os_disk_file_path,
+                size_gib=node_runbook.disk_img_resize_gib,
+            )
```

Comment:
You should resize after the below cp or qemu-convert operation. Otherwise `os_disk_file_path` is non-existent path.

Reply:
using os_disk_file_path in _create_node_os_disk() after conver/copy under CH platform and after create_diff_qcow2() for QEMU platform